workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -445,7 +439,6 @@ def composer_db_schema_get_current_revision():
     if version['edition'] != 'premium':
         return None
     output = subprocess.check_output([
-        'sudo', '-u', snapshot_constants.COMPOSER_USER,
```

Comment:
I think these sudos are still needed, otherwise you don't haver permissions to access the migration.js, (and everything it accesses) as cfyuser (which the snapshot restore runs as).

Reply:
The same here - I don't recall removing those lines. Reverting.